lib/ansible/modules/clustering/pacemaker_docker.py

```diff
@@ -0,0 +1,300 @@
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: pacemaker_docker
+short_description: Manage docker resources in a pacemaker cluster
+version_added: "2.5"
+author: "Chafik Belhaoues (@africanzoe)"
+description:
+   - This module can create|delete|manage|unmanage a docker resource within a
+     pacemaker cluster using the pacemaker "pcs" cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the resource
+      choices: ['created', 'deleted', 'managed', 'unamanged']
+      required: true
+      default: created
+    resource:
+      description:
+        - Specify the name of the resource to be created.
+      required: true
+      default: None
+    docker_name:
+      description:
+        - Specify the name of the docker to be created, this name will appear
+          in "docker ps" command.
+      required: true
+      default: resource
+    group:
+      description:
+        - Specify the name of the resource group to be created.
+      required: false
+    after:
+      description:
+        - Specify the position of the added resource relatively to some resource
+          already existing in the group.
+      required: false
+    before:
+      description:
+        - Specify the position of the added resource relatively to some resource
+          already existing in the group.
+      required: false
+    image:
+      description:
+        - Specify the name of the image used to create the docker.
+      required: true
+    command:
+      description:
+        - Specify the command to execute inside the container.
+          Please consider using docker Entrypoints or CMD instead.
+      required: false
+    run_opts:
+      description:
+        - Specify options to be appended to the 'docker run' command which is
+          used when creating the container during the start action.
+      required: false
+    mount_points:
+      description:
+        - A comma separated list of directories that the container is expecting
+          to use. The agent will ensure they exist by running 'mkdir -p'.
+      required: false
+    monitor_cmd:
+      description:
+        - Specifiy the full path of a command to launch within the container to
+          check the health of the container.
+      required: false
+    timeout:
+      description:
+        - Timeout when the module should considered that the action has failed
+      required: false
+      default: 300
+    force_kill:
+      description:
+        - Specify wether to kill a container immediately rather than waiting
+          for it to gracefully shutdown.
+      required: false
+      default: true
+    allow_pull:
+      description:
+        - Specify wether to allow the image to be pulled from the configured
+          docker registry when the image does not exist locally.
+      required: false
+      default: true
+requirements:
+    - "python >= 2.6"
+'''
+EXAMPLES = '''
+---
+# Create a  docker resource
```

Comment:
This line can be removed

Reply:
I didn't get which line needs to be removed?

Reply:
The `#` is a repeat of the `- name:`
Therefore the comment can be deleted.